### PR TITLE
Feature/experiment list ordering

### PR DESCRIPTION
**Pull request by jluethi**
Improves the view of experiments in the overview (switch to reverse chronological ordering)

Also includes 2 other fixes:
1. Makes the combine channels module more general & more useful
2. Increases init memory a bit to avoid edge cases.

### CODE REVIEW

**Comment by jluethi** [comment]
I updated the memory requirements for init & collect to the highest value needed for any user so far.
As both those jobs are normally very fast, even high memory requirements shouldn't be an issue and should decrease the support load when issues arise with large experiments

File: jtlibrary/python/jtmodules/src/jtmodules/combine_channels.py [hunk 1]
```diff
@@ -75,13 +65,15 @@ def main(image_1, image_2, weight_1, weight_2, plot=False):
...
-    combined_image = img_1 * weight_1 + img_2 * weight_2
+    combined_image = image_1 * weight_1 + image_2 * weight_2
+    # Set all values below 0 to 0
+    combined_image[combined_image < 0] = 0
+    combined_image[combined_image > max_val] = max_val
     logger.info('cast combined image back to correct data type')
-    combined_image = mh.stretch(combined_image, 0, max_val, image_1.dtype)
+
+    # Cast image to the same type as the input image
+    combined_image = combined_image.astype(image_1.dtype)
```
Comment (scottberry) [inline jtlibrary/python/jtmodules/src/jtmodules/combine_channels.py:76]: I actually implemented this for myself already, sorry for not merging! I did it like this:
```
    logger.debug('cast images to type float for arithmetics')
    img_1 = image_1.astype(np.float32) / max_val
    img_2 = image_2.astype(np.float32) / max_val
    logger.debug('combine images using the provided weights')
    combined_image = (img_1 * float(weight_1) +
                      img_2 * float(weight_2)) * max_val
    combined_image[combined_image > max_val] = max_val
    logger.debug('cast combined image back to correct data type')
    combined_image = combined_image.astype(image_1.dtype)
```
which I think is the same?? But I only allowed for positive weights

**Review by scottberry — COMMENTED** [review]
just check the comments I left in `combine_channels`

File: jtlibrary/python/jtmodules/src/jtmodules/combine_channels.py (hunk 1 above)
Comment (jluethi) [reply jtlibrary/python/jtmodules/src/jtmodules/combine_channels.py:76]: @scottberry Yes, that looks like it's doing the same. I would like to keep negative values though. I found this useful, especially for simplistic pipelines, e.g. segmenting nuclei ok by just subtracting Lamin from DAPI and thresholding the result (not as good as segmenting the envelope, but quick & effective)

**Comment by jluethi** [comment]
@scottberry Thanks for the review! :) Please set to approved afterwards so that we can merge it into master later :)

**Review by bebosudo — APPROVED** [review]

**Review by scottberry — APPROVED** [review]

**Pull request merged**